tests/baselines/reference/noImplicitAnySpreads1.errors.txt

```diff
@@ -0,0 +1,22 @@
+noImplicitAnySpreads1.ts(13,3): error TS7018: Object literal's property 'other' implicitly has an 'any' type.
+
+
+==== noImplicitAnySpreads1.ts (1 errors) ====
+    // https://github.com/microsoft/TypeScript/issues/58150#issuecomment-2052517378
+    
+    function getMore() {
+      return {
+        c: "foo",
+        p: "bar",
+        s: "baz",
+      };
+    }
+    
+    const bar = {
+      p: null, // no error, this gets overriden
+      other: null, // error, this does not get overriden

```

Comment:
I am confused. This is the behaviour I get when I paste the test into the playground and turn off strictNullChecks. What does this PR fix about this test?

Also, this isn't the case mentioned in the linked comment, as far as I can tell.

Reply:
To be honest... I'm confused too. I've checked out previous commits, re-run this test and all and it doesn't make a lot of sense to me why I have added this here.

**Maybe** this is a test case validating that I didn't break this behavior with my changes and I forgot to add a test case from the referenced comment? That case from the comment **does** exhibit a change. I just added it to the test file here and u can compare it with the current non-sensical behavior [here](https://www.typescriptlang.org/play/?strictNullChecks=false&ts=5.6.0-dev.20240615#code/PTAEAEGcBcCcEsDG0ByBXANhgwgCwKaIDWkAXKAGYCGGk+AUCBAHYD2AkgLYAOGS80AILMAnuThoGFNM2TxWzUABNWAZWi4A6gNwAFKrCqdIACm4GjZUDKJsA7swCUoAN71QoWPmhpYitx4eiOQA5BSsrCEANO6B3KEARgbRsR5WIUkAXikeAL4A3PS59PSICjCgWaAAvK6x8aDMmBgxaeRNWK2gAHS9KupaOvqGxiYBcaEEWJFdbaAhdqywGEo5oLmOMQVAA)